app/views/users/_privacy_settings.haml

```diff
@@ -27,9 +27,7 @@
           - if @blocks.length.zero?
             %p
               = t('.no_user_ignored_message')
-
-          - @blocks.each do |block|
-            = block.person_name
-            \-
-            = link_to t('.stop_ignoring'), block_path(block),
-              method: :delete
+          - else
+            #blocked_people
+              - for block in @blocks
+                = render :partial => 'blocked_person', :locals => {block: block, person: block.person}
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.